Marlin/Configuration.h

```diff
@@ -550,6 +550,27 @@
 #define Z_MAX_ENDSTOP_INVERTING false // set to true to invert the logic of the endstop.
 #define Z_MIN_PROBE_ENDSTOP_INVERTING false // set to true to invert the logic of the probe.
 
+/**
+ * Stepper Drivers for all axes
+ *
+ * Options: A4988, DRV8825, LV8729, TB6560, TB6600, TMC2100, TMC2130, TMC2208, TMC26X, TMC2660, TMC5130
+ * Add: DRIVER_USES_SPI when SPI is used (eg #define X_DRIVER_TYPE TMC2130 + DRIVER_USES_SPI)
+ * If all undefined, DRV8825 is assumed.
+*
+ * :['A4988', 'DRV8825', 'LV8729', 'TB6560', 'TB6600', 'TMC2100', 'TMC2130', 'TMC2208', 'TMC26X', 'TMC2660', 'TMC5130']
+ */
+//#define X_DRIVER_TYPE
+//#define X2_DRIVER_TYPE
+//#define Y_DRIVER_TYPE
+//#define Y2_DRIVER_TYPE
+//#define Z_DRIVER_TYPE
+//#define Z2_DRIVER_TYPE
+//#define E0_DRIVER_TYPE
+//#define E1_DRIVER_TYPE
+//#define E2_DRIVER_TYPE
+//#define E3_DRIVER_TYPE
+//#define E4_DRIVER_TYPE
```

Comment:
I think these should all have the default value of DRV8825, but still remain commented out, since that is the default and it makes it clear that these must be defined as one of the listed options, and are not just on/off switches.